### DISCUSSION TITLE
Nuxt 3 Plugin

### DISCUSSION THREAD

**Discussion by productdevbook** [urql-graphql/urql#2997] [Q&A]
https://github.com/productdevbookcom/awesome-nuxt-plugins

https://github.com/productdevbookcom/awesome-nuxt-plugins/blob/main/v3/urql.ts

I added it here.

**Comment by kitten** [comment]
Quick question :wave: Do you think this is generic enough for us to make a `@urql/nuxt-plugin` package, similar to what we provide for `next-urql`?
This section seems to be very generic to me at least: https://github.com/productdevbookcom/awesome-nuxt-plugins/blob/d4768f66ca5e57d8b95e0bbc747778e2294d180c/v3/urql.ts#L27-L39

**Reply by productdevbook** [reply]
Actually, I wanted to write a library and send a pr, but it will be a bit strange to set up an npm library for those two codes. Apart from this, if there will be additional features and large structures, it would be more appropriate to do npm.
